src/pages-and-resources/discussions/app-list/data/slice.js

```diff
@@ -21,11 +21,15 @@ const slice = createSlice({
   reducers: {
     loadApps: (state, { payload }) => {
       state.activeAppId = payload.activeAppId;
+      // When the UI loads, we want to set the selectedAppId to the activeAppId.  This ensures the
+      // active one will be checked.
+      state.selectedAppId = payload.activeAppId;
```

Comment:
This was missing and would have caused issues if we send an activeAppId up from the backend - the card for that app wouldn't have been selected.